pkg/crypto/cryptoservices/cryptoservices.go

```diff
@@ -35,13 +35,13 @@ type Network interface {
 	EncryptJoinAccept(ctx context.Context, dev *ttnpb.EndDevice, version ttnpb.MACVersion, payload []byte) ([]byte, error)
 	EncryptRejoinAccept(ctx context.Context, dev *ttnpb.EndDevice, version ttnpb.MACVersion, payload []byte) ([]byte, error)
 	DeriveNwkSKeys(ctx context.Context, dev *ttnpb.EndDevice, version ttnpb.MACVersion, jn types.JoinNonce, dn types.DevNonce, nid types.NetID) (NwkSKeys, error)
-	GetNwkKey(ctx context.Context, dev *ttnpb.EndDevice) (types.AES128Key, error)
+	GetNwkKey(ctx context.Context, dev *ttnpb.EndDevice) (*types.AES128Key, error)
```

Comment:
Add comment that the returned key may still be `nil` even if the returned error is `nil`. Also below.